recipes-wpe/wpeframework/wpeframework-plugins-rdk_git.bbappend

```diff
@@ -0,0 +1,4 @@
+PACKAGECONFIG_append = " messenger packager"
+
+# Raspberry PI 3 has 4 cores, set to 4 for RPIs
+WPE_WEBKITBROWSER_THREADEDPAINTING = "2"
```

Comment:
Can we have a logic with bb.util.contains for rpi3 and rpi4 here?

Reply:
Will add

Reply:
done